dipy/workflows/multi_io.py

```diff
@@ -253,3 +253,10 @@ def __iter__(self):
         IO = np.concatenate([I, O], axis=1)
         for i_o in IO:
             yield i_o
+
+    @staticmethod
+    def file_existence_check(args):
+        input_args = list(args)
+        for input_path in sorted(input_args):
+            if len(glob(input_path)) == 0:
+                raise OSError('File not found: '+input_path)
```

Comment:
- Can you replace `OSError` by `IOError`?
- pep8: Can you add space around the operator `+`

Reply:
Yes, this is a logical suggestion. Thank you for pointing this out. I have replaced the OSError by IOError. 

I used OSError because the file/directory related errors/exceptions are placed inside the OSError on the python's exception page here https://docs.python.org/3/library/exceptions.html#os-exceptions.

I have changed the OSError to IOError. :)